python/dolfinx/fem/dofmap.py

```diff
@@ -56,3 +66,21 @@ def index_map_bs(self):
     def list(self):
         """Adjacency list with dof indices for each cell."""
         return self._cpp_object.map()
+
+
+def create_dofmaps(
+    comm: Comm, topology: "dolfinx.mesh.Topology", elements: Sequence[FiniteElement]
+) -> list[DofMap]:
+    """Create a set of dofmaps on a given topology
+
+    Args:
+        comm: MPI communicator
+        topology: Mesh topology
+        elements: Sequence of coordinate elements
+
+    Returns:
+        List of new DOF maps
```

Comment:
Need explain what is returned, e.g. "List of degree-of-freedom maps where the ``i``th map is the map for ``elements[i]``.